alchemist/alchemist-ns3/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2010-2019, Danilo Pianini and contributors listed in the main(project"s alchemist/build.gradle file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution"s top directory.
+ */
+
+dependencies {
+    api("com.github.gscaparrotti:ns3asy-bindings:0.3.0")
```

Comment:
run `./gradlew buildSrc`, and refer to the statically generated reference via `Libs.ns3asy` (or any way the system decides to call it).